game/src/pregame.rs

```diff
@@ -360,14 +360,15 @@ impl Proposals {
                 if edits.proposal_link.is_some() {
                     current_tab.push(
                         ctx.style()
-                            .btn_solid_text("Read detailed write-up")
```

Comment:
The link for this currently 404's. Maybe related to the org move?

https://dabreegster.github.io/abstreet/case_studies/west_seattle.html
https://dabreegster.github.io/abstreet/case_studies/lake_wash.html

Want me to file an issue?

Reply:
I think in general we want to style hyperlinks as underlined text like this:

<img width="903" alt="Screen Shot 2021-02-26 at 11 20 38 AM" src="https://user-images.githubusercontent.com/217057/109345312-c984a000-7824-11eb-9f60-bead4b2a0341.png">